cmd/anonymizer/app/anonymizer/anonymizer_test.go

```diff
@@ -98,15 +98,18 @@ func TestAnonymizer_Hash(t *testing.T) {
 }
 
 func TestAnonymizer_AnonymizeSpan_AllTrue(t *testing.T) {
+	options := &Options{
+		HashStandardTags: true,
+		HashCustomTags:   true,
+		HashProcess:      true,
+		HashLogs:         true,
+	}
 	anonymizer := &Anonymizer{
 		mapping: mapping{
 			Services:   make(map[string]string),
 			Operations: make(map[string]string),
 		},
-		hashStandardTags: true,
-		hashCustomTags:   true,
-		hashProcess:      true,
-		hashLogs:         true,
+		options: options,
```

Comment:
nit: you could just embed the struct initialization here, having separate variable does not improve readability